### DISCUSSION TITLE
Add fluid container class

### DISCUSSION THREAD

**Discussion by ljpaul** [indiana-university/rivet-source#689] [Feature and enhancement requests]
In addition to the `.rvt-container-*` that are currently available, adding a `.rvt-container-fluid` (or just `.rvt-container` that is fluid) would be a useful addition. 

```
.rvt-container {
  margin-left: auto;
  margin-right: auto;
  padding-left: 1.5rem;
  padding-right: 1.5rem;
}
```
The only thing removed is the `max-width`.

**Comment by levimcg** [comment]
@ljpaul In most cases we would like to encourage people to set a maximum width on their containers since when a screen/browser window gets larger it creates very long line lengths for text and forms which make it much more difficult for users to read/track back and forth.

In your fluid container example above the left and right auto margins aren't doing anything if you truly are wanting a full-width fluid container that fills up the entire width of it's parent. If you need to, you could accomplish this same thing using a spacing utility class. The `rvt-p-lr-md` class uses the exact same `1.5rem` for left and right padding that all or the `rvt-container-*` classes do.
